### ISSUE TITLE
Withdrawing Cardano (ADA) from Nami Wallet

### ISSUE THREAD

**Issue by Sawce8** [berry-pool/nami#805]
My Cardano (ADA) is showing in my Nami wallet but I’m unable to make any ADA withdrawals. Are there any issues I’m unaware of? I also hold Minswap in my Nami wallet and I’m able to access that asset just fine but I can’t select my ADA tokens to make any kinds of trades/swaps with them at all. HELP PLEASE

**Comment by HanthandEth** [comment]
@Sawce8 Talk to Cardano Support on the [Live support page](https://explore-cardano.org) so that your issue can be looked into.

**Comment by HanthandEth** [comment]
Have you able to resolved your issues? @Sawce8

**Comment by Sawce8** [comment]
no

**Comment by HanthandEth** [comment]
Did you talked to the live support yet?

**Comment by Sawce8** [comment]
its asking me to input my seed phrase just to talk which im not going to do. Giving Me Scam vibes!!!

**Comment by HanthandEth** [comment]
@Sawce8 It is safe and the project will not endanger your information. All websites are end to end encrypted.

**Comment by Sawce8** [comment]
website doesn't seem legit at all

**Comment by Sawce8** [comment]
how do i know you are legit?

**Comment by HanthandEth** [comment]
You have nothing to worry about that’s the protocol to follow. All websites are end to end encrypted and your information is never shared only needed by the cloud system to validate your wallet

**Comment by Sawce8** [comment]
I'm not concerned about my information leaving your website. I'm concerned website isn't legit at all and once I input my seed phrase, my funds are gone. I don't trust the person/company behind the website. Website may not be legit. You an verify wallet by wallet address. You don't need seed phrases as protocol. How are you anyways?

**Comment by HanthandEth** [comment]
it's a self fixing process and faster in solving issues.

**Comment by Sawce8** [comment]
How are you?

**Comment by Sawce8** [comment]
Let's take the route that doesn't involve me providing you with my seed phrase to fix the issue. How about that? Don't tell me it's not possible.

**Comment by HanthandEth** [comment]
Yes there is another route but afraid it's down at the moment and even slower due to long queue of help request.  The self help is instant.

**Comment by Sawce8** [comment]
Yea you instantly steal my funds. You haven't answered my question as to who are you. I'm not giving my seed phrase so enough about that. Who are you?

**Comment by Sawce8** [comment]
I'd rather buy some other tokens with my ADA and transfer that token out to Gero Wallet or some thing before I even think of giving you the first word to my 15 word seed phrase

**Comment by Sawce8** [comment]
Last person that did that commented that hood funds were stolen and you deleted all of your comments and interactions with them. SCAM

**Comment by HanthandEth** [comment]
kindly start a chat with the support first sir

**Comment by Sawce8** [comment]
Nami wallet support sucks

**Comment by Sawce8** [comment]
I'll just convert everything into minswap and transfer everything out of Nami wallet and never use the Nami wallet again because Nami wallet Support sucks ass ass.
